CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 One of the easiest ways to contribute is to participate in discussions on GitHub issues. You can also contribute by submitting pull requests with code changes.
 
+Here are videos where members of the AspNetCore Team provide guidance, advice and samples on how to contribute to this project.
+* For ASP.NET Core - https://www.youtube.com/watch?v=hVdwb41FPvU
```

Comment:
Thanks for your PR, @divyeshio.
The only concern I have with referencing recordings from documentation is that they're set in stone and can't be updated. I'm not even sure if the Blazor guidance in that video valid any more.

@captainsafia what do you think?

Reply:
I think the video is fine to add because it provides some additional context into how the build is setup. That being said, I would probably add some clearer messaging that the video might not reflect the most up-to-date information about how to set up the repo and the doc is the most authoritative doc.

Reply:
So, according to you, what part might be out of date and where can I find most up to date docs for it?

Reply:
We try to keep this document up to date.

Maybe following @captainsafia's suggestion here and adding an `Additional Resources` section in the bottom of the document with links there and a text saying "Here are some recordings (partially outdated) for how to set the repo up locally".